flexmeasures/ui/templates/crud/asset.html

```diff
@@ -122,40 +122,43 @@ <h3>Edit {{ asset.name }}</h3>
                                             disabled></input>
                                     </div>
                                 </div>
-                                <p>Fields below can form part of this asset's flex context (<a href="https://flexmeasures.readthedocs.io/en/latest/features/scheduling.html#the-flex-context">flex context</a>). They can also be set in a parent asset or in the API requests.</p>
-                                {% if asset_form.consumption_price_sensor_id %}
-                                    <div class="form-group">
-                                        {{ asset_form.consumption_price_sensor_id.label(class="col-sm-3 control-label") }}
-                                        <div class="col-sm-3">
-                                            {{ asset_form.consumption_price_sensor_id(class_="form-control") }}
-                                            {% for error in asset_form.errors.consumption_price_sensor_id %}
-                                            <span style="color: red;">[{{error}}]</span>
-                                            {% endfor %}
+                                <div class="form-group" style="border: 2px solid rgb(169, 176, 176); padding: 10px; margin-bottom: 10px;">
```

Comment:
I'd rather see this styling in flexmeasures.css.
Feel free to add another class for this, e.g. `form-group-bordered`.